ci/tox/Dockerfile

```diff
@@ -29,9 +29,27 @@ ENV PYTHON_VERSION=${python_version}
 ENV PYTHON=python${python_version}
 RUN apk add --no-cache gcc musl-dev # both packages are required to install mypy
 
+FROM base AS CIRCLE_CI
 RUN $PYTHON -m pip install tox
 ENV PATH=$PATH:/root/.local/bin/
```

Comment:
Nit: is this still needed? does `pip` install stuff in `~/.local/bin` even when `--user` mode is NOT active? 🤔

Reply:
Yeah, it crashes without it:
```
Installing git-machete script to /root/.local/bin

Installed /root/.local/lib/python3.10/site-packages/git_machete-3.7.2-py3.10.egg
Processing dependencies for git-machete==3.7.2
Finished processing dependencies for git-machete==3.7.2
+ git machete --version
git: 'machete' is not a git command. See 'git --help'.
```

Reply:
https://app.circleci.com/pipelines/github/VirtusLab/git-machete/1493/workflows/496f166b-54af-41cb-b9ee-2615fc80b2d4/jobs/6037